virt_lightning/virt_lightning.py

```diff
@@ -661,8 +671,18 @@ def name(self):
 
     @name.setter
     def name(self, name):
+        self.user_data["name"] = name
         self.dom.rename(name, 0)
 
+    @property
+    def fqdn(self):
+        return self.get_metadata("fqdn")
+
+    @fqdn.setter
+    def fqdn(self, value):
+        self.user_data["fqdn"] = value
```

Comment:
Before sign value to user_data need validate fqdn. FQDN must be not None and valid name.

Reply:
Done